### PR TITLE
Fix flood mode on air alarms incorrectly setting pressure bounds

### PR DESCRIPTION
**Pull request by neersighted**
:cl:
fix: Malf AI's flood ability now correctly sets vent pressure bounds -- the results are devastating... Try it!
/:cl:

### CODE REVIEW

File: code/modules/atmospherics/machinery/airalarm.dm [hunk 1]
```diff
@@ -412,110 +412,107 @@
...
 				send_signal(device_id, list(
-					"power"=0
+					"power" = 0
 				))
 			for(var/device_id in alarm_area.air_vent_names)
 				send_signal(device_id, list(
-					"power"= 1,
-					"checks"= 0,
+					"power" = 1,
+					"checks" = 2,
```
Comment (neersighted) [inline code/modules/atmospherics/machinery/airalarm.dm]: This is the real changes, the others were whitespace

**Comment by KazeEspada** [comment]
Can we make this not Malf Only? Sometimes as an atmos tech, flooding the room with air would be awesome.

**Comment by CPTANT** [comment]
It should either be a standard feature (you still have to go to every alarm individually) or you should be able to set the checks for all vents connected to an alarm at the same time.

Now you just have to uncheck 5 external checks for each alarm to flood a room.

**Comment by GunHog** [comment]
Emagging unlocks flood mode. Also, "refill" mode works just fine, if not better, than "flood" mode in many cases with plasma. Once you set the plasma alight, the intense pressure will shut off the vents, preserving the rest of your plasma for flooding in other areas.

NOTE: Do not do this with burn mix, as it will melt the floors, resulting in constant leakage and plasma waste. As a rogue AI, you must conserve your precious plasma reserves, as the crew can easily stop you with one unwrenched pipe.

**Comment by neersighted** [comment]
@CPTANT Modes are literally presets. This one sets the internal pressure check to zero.

**Pull request merged**